core/lib/protobuf_config/src/proto/utils.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package zksync.config;
+
+message Prometheus {
+  optional uint32 listener_port = 1; // required
+  optional string pushgateway_url = 2; // required
+  optional uint64 push_interval_ms = 3;
```

Comment:
nit: I'd use seconds here, we definitely don't want to push more often than 30s.

Reply:
https://github.com/matter-labs/zksync-era/blob/ec6e923ddb5063d391c59e04dad28f9d2c15dcd1/core/lib/config/src/configs/utils.rs#L12 specifies this in ms. I converted it 1-1 so that migration from env vars is straightforward. 

We can improve the config format later. For all duration params I'd suggest to use a Duration message:
https://github.com/matter-labs/era-consensus/blob/dbdc5312fa60d5f1716a6ac791218353c0cc451e/node/libs/protobuf/src/proto/std.proto#L23
or (if that's too verbose) a string parseable to time::Duration (like "10s", or "500ms"), so that the unit is not specified in the field name.

Reply:
SGTM